cfn-resources/cloud-backup-restore-jobs/mongodb-atlas-cloudbackuprestorejobs.json

```diff
@@ -158,7 +154,6 @@
     "/properties/DeliveryUrl",
     "/properties/Cancelled",
     "/properties/Failed",
-    "/properties/CreatedAt",
```

Comment:
Would make use of the changelog that was created for this resource to mention this removal and the reasoning. On another hand I see this attribute is present in terraform, we can create a ticket to deprecate the field there.